python/legate_dataframe/cmake/Modules/LinkPyarrowHeaders.cmake

```diff
@@ -15,26 +15,35 @@ include_guard(GLOBAL)
 
 find_package(Python REQUIRED COMPONENTS Development NumPy)
 
-execute_process(
-  COMMAND "${Python_EXECUTABLE}" -c "import pyarrow; print(pyarrow.get_include())"
-  OUTPUT_VARIABLE PYARROW_INCLUDE_DIR
-  ERROR_VARIABLE PYARROW_ERROR
-  RESULT_VARIABLE PYARROW_RESULT
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+# Add pyarrow and numpy includes to each target
+function(link_to_pyarrow_headers targets)
+  execute_process(
```

Comment:
I modified this function because it failed when called a second time. I needed to regenerate the PYARROW_INCLUDE_DIR variable on the second call.

Reply:
Probably vendored from somewhere, but the changes look fine and easy enough, thanks.